### PR TITLE
[ODS-4907] Fix issue with long path names needed for log4net in DbDeploy

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1 [hunk 1]
```diff
@@ -9,6 +9,7 @@ $ErrorActionPreference = 'Stop'
 
 $script:deploymentSettingsFiles = @()
 $script:deploymentSettingsOverrides = @{ }
+Import-Module -Force -Scope Global "$PSScriptRoot/Ed-Fi-ODS-Implementation/logistics/scripts/modules/path-resolver.psm1"
```
Comment (tbmeador-dlp) [inline Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1]: wouldn't importing `path-resolver` here prevent you from setting `$env:toolsPath` in your code below?

File: Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1 [hunk 2]
```diff
@@ -9,6 +9,7 @@ $ErrorActionPreference = 'Stop'
 
 $script:deploymentSettingsFiles = @()
 $script:deploymentSettingsOverrides = @{ }
+Import-Module -Force -Scope Global "$PSScriptRoot/../../../logistics/scripts/modules/path-resolver.psm1"
```
Comment (tbmeador-dlp) [inline Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1:12]: wouldn't importing `path-resolver` here prevent you from setting the `$env:toolsPath` below?

File: Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1 (hunk 1 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1]: OK fixed it . Thanks

File: Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1 (hunk 2 above)
Comment (semalaiappan) [reply Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1]: OK fixed it . Thanks

File: logistics/scripts/modules/tools/ToolsHelper.psm1 [hunk 3]
```diff
@@ -243,7 +243,7 @@ function Invoke-DbDeploy {
         [AllowEmptyCollection()]
         $Features = @(),
 
-        [string] $ToolsPath = (Get-ToolsPath)
```
Comment (tbmeador-dlp) [inline logistics/scripts/modules/tools/ToolsHelper.psm1:246]: why change this? doesn't `Get-ToolsPath` do the same check as below?

Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1:246]: Either case for initdev and deployment , I will have $env:toolsPath value and local variable $ToolsPath will be empty . so I am removing that default value assignment here .

Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1:246]: $env:toolsPath = $toolsPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'tools')  this line is exists in InitializeDevelopmentEnvironment.psm1

Comment (semalaiappan) [reply logistics/scripts/modules/tools/ToolsHelper.psm1:246]: $env:toolsPath = (Join-Path (Get-RootPath) 'tools') for deployment.psm1   https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/pull/266/files#diff-2774a29b696c69083c03917a3ee108d0cee3ae198c6526db3444b15be11421dcR81

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**